lib/errors.js

```diff
@@ -421,6 +426,22 @@ function createInvalidPluginImplementationError(
   return err;
 }
 
+/**
+ * Creates an error object to be thrown when file is unparsable
+ *
+ * @public
+ * @param {string} message - Error message to be displayed.
+ * @param {string} filename - File name
+ * @param {string} [reason] - Why the filename is unparsable.
+ */
+function createUnparsableFileError(message, filename, reason) {
```

Comment:
I think we should remove `reason`, given my comments above in `lib/cli/options.js`